spec/services/linked_data/graph_service_spec.rb

```diff
@@ -139,39 +121,56 @@
         end
       end
 
-      context 'when multiple languages passed in as strings in an array' do
-        let(:language) { ["en", "fr"] }
+      context "when mix of language markers on graph statements" do
+        let(:nomarker_de_buttermilk) { RDF::Literal.new("Buttermilch") }
+        let(:language) { [:fr] }
 
-        it 'returns the graph with English and French subset of triples' do
-          expect(subject.has_object?(en_dried_milk)).to be true
-          expect(subject.has_object?(en_buttermilk)).to be true
-          expect(subject.has_object?(en_condensed_milk)).to be true
+        before do
+          stub_request(:get, 'http://authority.with.language/search?query=foo')
+            .to_return(status: 200, body: webmock_fixture('lod_lang_search_filtering.nt'), headers: { 'Content-Type' => 'application/n-triples' })
+        end
```

Comment:
This mock includes data for each of the new conditions to be sure language is processed correctly for each condition.